app/views/users/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Hello from new.html.erb</h1>
+
+<%# <form accept-charset="UTF-8" action="/users" method="post"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+  <%# <label for="username"> Input your username </label><br>
+  <input type="text" id="username" name="user[username]">
+  <br><br>
+  <label for="email"> Input your e-mail </label><br>
+  <input type="email" id="email" name="user[email]">
+  <br><br>
+  <label for="password"> Input your Password </label><br>
+  <input type="password" id="password" name="user[password]">
+  <br><br>
+  <input type="submit" value="submit"> %>
+<%# </form> %>
```

Comment:
- [ ] You should be using the `form_authenticity_token` method.
Screenshot from Odin 👇 
![Screen Shot 2020-11-04 at 1 05 53 AM](https://user-images.githubusercontent.com/48804258/98053871-e4787200-1e39-11eb-85be-e6bb71f18dd2.png)
```ruby
<input
    type="hidden"
    name="authenticity_token"
    value="<%= form_authenticity_token %>"
  />
```

Reply:
We did use the authenticity token in our form, in the first line inside the form!

Reply:
Sorry! I must have missed that 🙈